decentraland/unity-renderer
# refactor: delayed api call, remove unused UI, bug fixing

## What this PR includes?

- Delayed the API call of the catalog to when the user uses the builder project panel with a jump in click, editor click or try to enter with the shortcut in the land
- Removed the UI of initial panel since now it has been replaced with the BuilderProjectPanel
- Bug fix: If you quickly enter/exit the builder in world mode the audio get errors
- Bug fix: If you quickly enter/exit the builder in world mode, the UI will disappear and skybox 
- Refactor: While we are loading the builder in world, we have eliminated the option to cancel the loading


## How to test it?

1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:refactor/biw-api-call&ENABLE_BUILDER_IN_WORLD
2. Enter in a scene and check all changes

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
